client/my-sites/importer/importer-blogger.jsx

```diff
@@ -48,14 +49,27 @@ class ImporterBlogger extends React.PureComponent {
 				}
 			),
 			uploadDescription: this.props.translate(
-				'Upload a {{b}}%(importerName)s export file{{/b}} to start importing into {{b}}%(siteTitle)s{{/b}}.',
+				'Upload a {{b}}%(importerName)s export file{{/b}} to start ' +
+					'importing into {{b}}%(siteTitle)s{{/b}}. If you want to know how to create the export file, ' +
+					'our {{inlineSupportLink/}} can help.',
 				{
 					args: {
 						importerName,
 						siteTitle: this.props.site.title,
 					},
 					components: {
 						b: <strong />,
+						inlineSupportLink: (
+							<InlineSupportLink
```

Comment:
So nice to see the fruits of our recent labours come together and be useful so quickly! This looks great :)

Reply:
It's a great feature, congrats Spen!